frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.html

```diff
@@ -180,38 +180,19 @@ <h5 class="fw-light">All Submissions</h5>
                       </strong>
                     </p>
                   </div>
-                  <div class="col-sm-4 col-md-4 col-lg-4">
-                    <p>
-                      <strong class="text-light-black fw-semibold fs-12"
-                        >Metadata file&nbsp;<i class="fa fa-file fs-12" aria-hidden="true"></i></strong
-                      >&nbsp;
-                      <a
-                        target="_blank"
-                        class="blue-text fs-12"
-                        [href]="element.submission_metadata_file"
-                        *ngIf="element.submission_metadata_file"
-                      >
-                        <i class="fa fa-external-link"></i> Link
-                      </a>
-                      <strong *ngIf="!element.submission_metadata_file" class="fw-semibold fs-12">
-                        None
-                      </strong>
-                    </p>
-                  </div>
                 </div>
               </div>
-
-              <div class="elements-action">
+              <div class="elements-status">
                 <div class="row rm-margin">
-                  <div class="col-sm-6 sol-md-6 col-lg-4 no-pad-lr">
+                  <div class="col-sm-6 sol-md-6 col-lg-5 no-pad-lr">
                     <p>
                       <strong class="text-light-black fw-semibold fs-12">Status</strong> &nbsp;
                       <strong class="fw-semibold fs-14" [ngClass]="element.status" class="fs-12">
                         {{ element.status | uppercase }}
                       </strong>
                     </p>
                   </div>
-                  <div class="col-sm-6 sol-md-6 col-lg-8 no-pad-lr">
+                  <div class="col-sm-6 sol-md-6 col-lg-7 no-pad-lr">
```

Comment:
@savish28 why are the column size inconsistent across divs? We should have equal size